docs/setup/independent/install-kubeadm.md

```diff
@@ -55,73 +56,109 @@ documentation for the plugins about what port(s) those need.
+* `kubectl`: the command line util to talk to your cluster.
 
-**Note:** If you already have kubeadm installed, you should do a `apt-get update &&
-apt-get upgrade` or `yum update` to get the latest version of kubeadm. See the
-kubeadm release notes if you want to read about the different [kubeadm
-releases](https://github.com/kubernetes/kubeadm/blob/master/CHANGELOG.md).
-
-For each machine:
-
-* SSH into the machine and become root if you are not already (for example,
-  run `sudo -i`).
-
-* If the machine is running Ubuntu or HypriotOS, run:
-
-  ``` bash
-  apt-get update && apt-get install -y apt-transport-https
-  curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-  cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
-  deb http://apt.kubernetes.io/ kubernetes-xenial main
-  EOF
-  apt-get update
-  apt-get install -y kubelet kubeadm
-  ```
-
-* If the machine is running CentOS, run:
-
-  ``` bash
-  cat <<EOF > /etc/yum.repos.d/kubernetes.repo
-  [kubernetes]
-  name=Kubernetes
-  baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
-  enabled=1
-  gpgcheck=1
-  repo_gpgcheck=1
-  gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
-          https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
-  EOF
-  setenforce 0
-  yum install -y kubelet kubeadm
-  systemctl enable kubelet && systemctl start kubelet
-  ```
+Please proceed with executing the following commands based on your OS as the `root` user by SSH-ing
+into each machine. You may for example become `root` by executing `sudo -i`.
 
-The kubelet is now restarting every few seconds, as it waits in a crashloop for
-kubeadm to tell it what to do.
+{% capture ubuntu %}
+
+```bash
+apt-get update && apt-get install -y apt-transport-https
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+apt-get update
+apt-get install -y kubelet kubeadm kubectl
+```
+
+{% endcapture %}
+
+{% capture centos %}
+
+```bash
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+        https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+setenforce 0
+yum install -y kubelet kubeadm
+systemctl enable kubelet && systemctl start kubelet
+```
 
 **Note:** Disabling SELinux by running `setenforce 0` is required to allow
-containers to access the host filesystem, which is required by pod networks for
-example. You have to do this until SELinux support is improved in the kubelet.
+containers to access the host filesystem, which is required for the `kubeadm init`
+process to complete successfully. You have to do this until SELinux support
+is improved.
```

Comment:
Would it be appropriate to link to the tracking issue here, for those who are interested? I'm not sure how often we do that from our user docs.

Reply:
I don't think right now is necessary. We haven't specced out a plan yet